html-syntax/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+  <title>HTML Syntax</title>
+</head>
+
+<body>
+  <img src="https://i.pinimg.com/originals/12/8d/e8/128de8ce51ee0c498a4dfa67610f5843.jpg" alt = "Snorlax">
```

Comment:
You don't want spaces around your equal signs when declaring attributes, it can mess up HTML's parsing in the browser